### PR TITLE
animations overview

### PR DESCRIPTION
**Pull request by rachelandrew**
Part of #3864 

Changes proposed in this pull request:

- Creates new folder for animations
- Adds animations.json
- First draft of "Why are some animations slow?"

### CODE REVIEW

File: src/site/content/en/animations/index.njk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: path
+override:tags: []
+date: 2020-09-25
+pathName: animations
+tags:
+  - animations
```
Comment (robdodson) [inline src/site/content/en/animations/index.njk:7]: ah we should remove these. They were causing a bug with our /tags/ landing page.

**Comment by kaycebasques** [comment]
Thanks, Rachel.

My first impression is that this overview is blurring the lines between explaining 1) why some animations are slow and 2) how to create high-performance animations.

[Divio's Documentation System](https://documentation.divio.com/) does a good job explaining why we want this "separation of concerns" as it were.

I would like the overview to focus on conceptual explanation of why animations are slow. In the beginning and the end of the overview we can say, "if you don't care about the theory, check out the guide"

And then we have a guide that focuses on how to create high-performance animations, with less explanation on why they're fast. This will optimize for the type of web developer who doesn't really care why, they just want to get it done. In the beginning or end we can say "if you want to understand the theory behind high-performance animations, check out the overview"

Logistical note: feel free to do all of the work related to the animations learning path within this PR (i.e. you don't need to create a separate PR for the guide)

**Comment by rachelandrew** [comment]
@kaycebasques I'm not sure about having a conceptual overview that doesn't tie this stuff back to what people know about. I don't think I'm over explaining the how, I'm giving one example (the very simple top left one) in order to tie some theory to something a reader might reasonably understand. As mostly the key thing to understand is where in the steps things happen, the top left example is slower because of the place in the process it touches.

I could remove the example, personally I'd rather keep it.

The forcing layers creation stuff could move.

I'll look at adding in your links.

I'll add to this PR the next article (the guide to creating animations) as perhaps it will be easier to figure out what goes where in the light of both.

**Comment by kaycebasques** [comment]
Work is continuing in #4019

**Pull request closed without merging**